### DISCUSSION TITLE
yeet

### DISCUSSION THREAD

**Discussion by aidan-crockett** [CluckCluckChicken/Magnifier#41] [big chungus]
:yeetgoat: 🐐 :yeetgoat: 🐐 :yeetgoat: 🐐 :yeetgoat: 🐐 :yeetgoat: 🐐 :yeetgoat: 🐐 :yeetgoat: 🐐 :yeetgoat: 🐐 
![image](https://user-images.githubusercontent.com/82768218/120081043-132d9280-c081-11eb-85db-29600459bd24.png)

**Comment by aidan-crockett** [comment]
yeet goat kinda sus

**Reply by aidan-crockett** [reply]
😳

**Comment by davwat** [comment]
this topic is called yeet and your pfp is nasa and stuff can be yeeted into space with i have no idea where im going with this

**Reply by aidan-crockett** [reply]
yeet goat will sort this out!!!11!11!11! :goat:
<img width="200" src="https://user-images.githubusercontent.com/82768218/120081043-132d9280-c081-11eb-85db-29600459bd24.png">

**Comment by JackyBorderCollie** [comment]
goatgoatgoat

**Comment by FunctionalMetatable** [comment]
heck how did yeetgoat come to github

**Reply by aidan-crockett** [reply]
you mentioned him on the @Basic8 topic

**Reply by FunctionalMetatable** [reply]
o

**Reply by aidan-crockett** [reply]
I found a higher res yeetgoat :goat:
https://covers.alphacoders.com/cover/view/162653
